config/pacbio_smrt_link_versions.yml

```diff
@@ -268,6 +275,36 @@ default: &default
         - v12_revio
         - v13_revio
         - v13_1_revio
+        - v25_1_revio
+    use_adaptive_loading:
+      key: use_adaptive_loading
+      label: "Use Adaptive Loading"
+      default_value: "False"
+      validations:
+        presence: {}
+        inclusion:
+          in: *true_false
+        use_adaptive_loading: {}
+      data_type: list
+      select_options: *select_true_false
+      versions:
+        - v13_revio
+        - v13_1_revio
+        - v25_1_revio
+    full_resolution_base_qual:
+      key: full_resolution_base_qual
+      label: "Full Resolution Base Qual"
+      default_value: "False"
+      validations:
+        presence: {}
+        inclusion:
+          in: *true_false
+      data_type: list
+      select_options: *select_true_false
+      versions:
+        - v13_revio
+        - v13_1_revio
+        - v25_1_revio
```

Comment:
Again to understand, I thought these two fields are added as part of v25. Are these two fields required in v13 as well?

Reply:
Same as previous comment.